index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>setup and mobile first</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <main>
+      <div class="illustration-head-line">
+        <!-- Starting Tool bar-->
+        <nav class="nav">
+          <div>
+            <h2 class="Subhead Neutral-1">Welcome</h2>
+          </div>
+          <div class="Humbger-nav">
+            <div class="Union">
+              <span class="Vector"></span>
+              <span class="Vector"></span>
+              <span class="Vector"></span>
+            </div>
+          </div>
+        </nav>
+        <!-- ending Tool bar-->
+        <!-- Starting Head Line-->
+        <div class="Head-Line">
+          <div class="container-Head-Line">
+            <!-- Starting Primary-->
+            <div class="Primary-Head-line ">
+              <h2 class="h2 Neutrals-N800 no-margin-bottom">
+                Hey there.I'm Ammar
+              </h2>
+              <h2 class="h2 Green-trust no-margin-top">
+                I'm a software developer
+              </h2>
+            </div>
+            <!-- ending Primary-->
+            <!-- Starting Supporting text-->
+            <div>
+              <p class="p Neutrals-N800">
+                I can help you build a product , feature or website Look through
+                some of my work and experience! If you like what you see and
+                have a project you need coded, don’t hestiate to contact me.
+              </p>
+            </div>
+            <!-- ending Supporting text-->
+          </div>
+        </div>
+        <!-- ending Head Line-->
+        <!-- Starting Social Media-->
+        <div class="social-media-container">
+          <div>
+            <a href="https://twitter.com/raezon_dev"><img src="img/twitter.svg" alt="twitter"/></a>
+          </div>
+          <div>
+            <a href="https://www.linkedin.com/in/ammar-djebabla-519030190/"><img src="img/Linkedin icon.svg" alt="Linkedin"/></a>
+          </div>
+          <div>
+            <a href="#"><img src="img/medium.svg" alt="medium"/></a>
+          </div>
+          <div>
+            <a href="https://github.com/raezon"><img src="img/github.svg" alt="github"/></a>
+          </div>
+          <div>
+            <a href="#"><img src="img/angellist.svg" alt="angellist"/></a>
+          </div>

```

Comment:
-Good job wrapping each icon in an `a` tag 👏 but when dealing with lists in HTML, it is semantically correct to wrap them in a `ul`  tag and wrap each `a` tag in a corresponding `li` tag. Please fix this 👍 👍